.devcontainer/devcontainer.json

```diff
@@ -16,6 +16,9 @@
         "nefrob.vscode-just-syntax",
         "redhat.vscode-yaml"
       ],
+      "files.readonlyInclude": {
+        "kustomization/components/kube-vip/daemonset.yml": true
+      },
```

Comment:
Why is this part necessary? Don't we want some way to be able to update this in the devcontainer when testing the new codegen script?

Reply:
The codegen script can change it all it wants.  It only makes it human un-editable (and you can disable it too).  It basically makes it more obvious that you shouldn't edit this by hand.